.bazelrc

```diff
@@ -256,8 +256,9 @@ common --experimental_profile_include_primary_output
 common --noslim_profile
 # Include compact execution log
 # TODO(sluongng): make Bazel writes this to output_base automatically
-common:linux --experimental_execution_log_compact_file=bazel_compact_exec_log.binpb.zstd
-common:macos --experimental_execution_log_compact_file=bazel_compact_exec_log.binpb.zstd
+common:linux --experimental_execution_log_compact_file=bazel_compact_exec_log.binpb.zst
+common:macos --experimental_execution_log_compact_file=bazel_compact_exec_log.binpb.zst
+common:windows --experimental_execution_log_compact_file=bazel_compact_exec_log.binpb.zst
```

Comment:
Does this still need to be platform-dependent?

Reply:
Good call. We can make it a generic flag now.